frontend/app/tests/e2e/pages/account-balances-page/blockchain-balances-page.ts

```diff
@@ -111,7 +112,7 @@ export class BlockchainBalancesPage extends AccountBalancesPage {
       .find('tr')
       .eq(position + (this.isGroupped(balance) ? 0 : 1))
       .find('button.actions__edit')
-      .click();
+      .click({ timeout: 240000 });
```

Comment:
It is ok as a temporary fix, but like we discussed we will need to adjust the tests since the test is supposed to verify that the balances also load and it doesn't do it now.